src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -102,10 +109,19 @@ public static LocalDateTime getFirstDateTimeMustBe() {
     public static int getHearingDuration(SscsCaseData caseData) {
         // TODO Adjournments - Check this is the correct logic for Adjournments
         // TODO Future Work - Manual Override
-        // TODO Dependant on SSCS-10116 - Will use Session Category Reference Data
 
+        int duration = getHearingDurationAdjournment(caseData);
```

Comment:
I'm not really a fan of this and the related changes, will think about it more tomorrow.